dashboard/views/addEditUser.ejs

```diff
@@ -14,7 +14,7 @@
                             <div class="col-md-12">
                                 <div class="form-group form-black label-floating">
                                     <label class="control-label" data-l10n-id="name"></label>
-                                    <input class="form-control" name="name" value="<% if(typeof user == 'object'){ %><%= user.name %><% } %>" type="text">
+                                    <input class="form-control" name="name" value="<% if(typeof user == 'object'){ %><%= user.name %><% } %>" type="text" required>
```

Comment:
I'm not in favor of using "required". It's not consistent with other forms plus it don't use the same localization than the server. You could just let the server do the work.

Reply:
I believe there is no way we can have validation on the server without showing the popup. The page needs to be reloaded and thus the popup shows.